### ISSUE TITLE
Docker documentation for those who cannot use the Dockerfile traditionally?

### ISSUE THREAD

**Issue by GlassedSilver** [KyubiSystems/Yomiko#13]
Hey!

So I'm running unRAID and anything I wanna add to it with dockers needs to be done in a little different way.

Basically what is required besides the Dockerfile is all necessary path and port mappings, variables, etc...

The Dockerfile leads me to believe that I have to map the port 5000 to the host for WebUI access, the path /usr/src/app to a host path of preference for appdata and set the shell to bash.

Networking I set to bridge, privileged execution is false and that's that I guess?

Maybe I should also define PUID and PGID?

So far the execution of the docker always stops. :/

**Comment by KyubiSystems** [comment]
I have several platforms which will run Docker, but I don't have unRAID, so I can't test it myself. Try setting PUID/PGID to the current external user. Use 'docker logs' to check the script output.

To be honest, I'd suggest just cloning the repo and running the code outside a container to see if that works for you. The Docker packaging was more of a "because I can" than a recommended method of distribution.

**Comment by GlassedSilver** [comment]
Nothing in that log at all. Maybe you can tell me if you see anything obvious missing from my docker run command:

`docker run -d --name='Yomiko' --net='bridge' -e TZ="Europe/Berlin" -e HOST_OS="Unraid" -e 'PUID'='99' -e 'PGID'='100' -p '5000:5000/tcp' -v '/mnt/user/appdata/yomiko/':'/usr/src/app':'rw' 'kyubi/yomiko:alpine' `

**Comment by KyubiSystems** [comment]
Okaaay, first time I've looked at the codebase in a while, and it looks like it hasn't survived 2to3 conversion intact :(. There are some module path issues I'm fixing as I go. and I'll upload new Docker images.

**Comment by GlassedSilver** [comment]
Many many thanks! Looking forward to trying out your application! :)

**Comment by GlassedSilver** [comment]
Any update on this? Would be really interested in giving this a spin on my setup.

**Comment by KyubiSystems** [comment]
Yeah, I've been "away" dealing with some health issues. Haven't abandoned this (honest), and will have more time to look at this in the next couple of weeks.
